src/utils.ts

```diff
@@ -127,6 +130,19 @@ export async function deployTokenFaucet(
   )
 }
 
+export function simulateInvariantSwap(
+  tickmap: Tickmap,
+  feeTier: _FeeTier,
```

Comment:
To avoid the hassle of converting from and to the Ralph version of the structure I mostly try to use directly what `await newFeeTier(...)` returns. Otherwise I would have to convert it inside of every SDK entrypoint that takes the structure in as a function parameter, and that's a bigger change that deserves its own PR. I will perform the "unwrapping" here as an example.